.gitignore

```diff
@@ -170,4 +170,7 @@ tmp/
 *.sqlite3
 
 /resources/rooms/images
-general.txt
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Merge tags here as anna pointed out, can you remove .gitignore and the migration and splitutils from this pr?